.github/workflows/autobuild.yml

```diff
@@ -6,20 +6,28 @@ on:
     tags:
       - "r*"
       - "latest"
-    #branches:
+    branches:     
+      - "autobuild*"
```

Comment:
Not sure about this, but is this a debugging branch?

Reply:
yes. developers can use branches with that pattern, to follow closely if any commit breaks their build

adding a comment to clarify

Reply:
Ok. Thanks.